tools/README.md

```diff
@@ -1,3 +1,10 @@
+About the dagmc_get_materials and test scripts
```

Comment:
It looks like `dagmc_get_materials` has been deleted in favor of `uwuw_preproc`. You still reference `dagmc_get_materials` numerous times in this pull request. Please update.